packages/frontend/core/src/modules/telemetry/services/telemetry.ts

```diff
@@ -2,27 +2,39 @@ import { mixpanel } from '@affine/track';
 import type { GlobalContextService } from '@toeverything/infra';
 import { ApplicationStarted, OnEvent, Service } from '@toeverything/infra';
 
-import { AccountChanged, type AuthAccountInfo, AuthService } from '../../cloud';
-import { AccountLoggedOut } from '../../cloud/services/auth';
-import type { ServersService } from '../../cloud/services/servers';
+import {
+  AccountChanged,
+  type AuthAccountInfo,
+  AuthService,
+  type DefaultServerService,
+} from '../../cloud';
 
 @OnEvent(ApplicationStarted, e => e.onApplicationStart)
-@OnEvent(AccountChanged, e => e.updateIdentity)
-@OnEvent(AccountLoggedOut, e => e.onAccountLoggedOut)
 export class TelemetryService extends Service {
   private readonly authService;
+
+  private readonly disposableFns: (() => void)[] = [];
+
   constructor(
-    serversService: ServersService,
-    private readonly globalContextService: GlobalContextService
+    private readonly globalContextService: GlobalContextService,
+    defaultServerService: DefaultServerService
   ) {
     super();
 
     // TODO: support multiple servers
-    const affineCloudServer = serversService.server$('affine-cloud').value;
-    if (!affineCloudServer) {
-      throw new Error('affine-cloud server not found');
-    }
-    this.authService = affineCloudServer.scope.get(AuthService);
+    this.authService = defaultServerService.server.scope.get(AuthService);
+
+    this.disposableFns.push(
+      defaultServerService.server.eventBus.on<AuthAccountInfo | null>(
+        AccountChanged,
+        account => this.updateIdentity(account)
+      )
+    );
+    this.disposableFns.push(
+      defaultServerService.server.eventBus.on(AccountChanged, () =>
+        this.onAccountLoggedOut()
+      )
+    );
   }
 
   onApplicationStart() {
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line +59]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->reset when switching to new user<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/8979?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->